### PR TITLE
Kiali 2478 Graph Find/Hide help improvements

### PR DESCRIPTION
**Pull request by jshaughn**
Applies the current suggestions from @jstickler and @abonas : 
- Add preface text
- Rename tips
- Rearrange tabs
- Update some of the existing help text

The result for feedback:
image

### CODE REVIEW

**Comment by jshaughn** [comment]
@jsticker, how does the above screenshot look to you?

**Comment by JStickler** [comment]
@jshaughn, nice job on the overview text.  Just a minor tweak for clarity, "You can use the Find and Hide filters to highlight or remove edges and nodes from the graph.  Each field accepts text expressions using the language described below."

File: src/pages/Graph/GraphHelpFind.tsx [hunk 1]
```diff
@@ -185,6 +185,11 @@ export default class GraphHelpFind extends React.Component<GraphHelpFindProps> {
 
   render() {
     const className = this.props.className ? this.props.className : '';
+    const preface =
+      'The Find and Hide input boxes are used to highlight or remove graph nodes and edges. Each accepts ' +
```
Comment (abonas) [inline src/pages/Graph/GraphHelpFind.tsx]: "remove" might sound scary :)
perhaps hide/not display?

File: src/pages/Graph/GraphHelpFind.tsx [hunk 2]
```diff
@@ -130,52 +130,52 @@ export default class GraphHelpFind extends React.Component<GraphHelpFindProps> {
...
-  operatorColumns = () => {
+  noteColumns = () => {
     return {
       columns: [
         {
-          property: 'o',
+          property: 't',
           header: {
-            label: 'Operator',
+            label: 'Usage Note',
```
Comment (abonas) [inline src/pages/Graph/GraphHelpFind.tsx]: note or notes in plural?

File: src/pages/Graph/GraphHelpFind.tsx (hunk 1 above)
Comment (JStickler) [reply src/pages/Graph/GraphHelpFind.tsx]: You're right, hide is better.

Comment (jshaughn) [reply src/pages/Graph/GraphHelpFind.tsx]: +1

File: src/pages/Graph/GraphHelpFind.tsx (hunk 2 above)
Comment (jshaughn) [reply src/pages/Graph/GraphHelpFind.tsx]: 'Usage Note' is correct here because it is the column header.  The tab is still 'Usage Notes'.

**Comment by jshaughn** [comment]
@JStickler @abonas , updated:

image

**Comment by jshaughn** [comment]
@abonas Anything else needed here or can we approve?

**Comment by abonas** [comment]
we can always improve more later in another PR :)

**Comment by jshaughn** [comment]
> we can always improve more later in another PR :)

@abonas I totally agree, so please approve and we can merge this in...

**Pull request merged**